tests/Tests/800__culler/culler.robot

```diff
@@ -105,8 +112,10 @@ Verify That "Stop Idle Notebook" Setting Is Not Overwritten After Restart Of Ope
     Oc Delete    kind=Pod     namespace=redhat-ods-operator    label_selector=name=rhods-operator
     sleep   5    msg=waiting time for the operator pod to be replaced with new one
```

Comment:
shouldn't the test wait until the operator pod is "running"? After 5 seconds, it will be in "init" state

Reply:
I think @tarukumar originally wrote this test; looking at the behaviour of the dashboard I don't see anything weird with the 5s sleep so it might be enough

Reply:
yeah I meant to say if we wait until the operator is in "running" state we can be more sure about culler configurations are not overwrittern

Reply:
@tarukumar any thoughts on this?

Reply:
ok, I think it is worth to wait 1-2 minutes